.gitmodules

```diff
@@ -17,9 +17,10 @@
 [submodule "stochastic_physics"]
 	path = stochastic_physics
 	url = https://github.com/noaa-psd/stochastic_physics
+	branch = master
 [submodule "MOM6"]
 	path = MOM6
-	url = https://github.com/NOAA-EMC/MOM6-interface
+	url = https://github.com/NOAA-EMC/MOM6-interface.git
```

Comment:
Why do we need to add .git in the url?

Reply:
The .git is there because I copied it from the "clone with HTTPS" on the GitHub interface when I changed this back from my fork.  It's not necessarily needed though I do not think.  If you want me to remove this just let me know.

Reply:
Let's remove it to be consistent with other urls.